jbmc/CMakeLists.txt

```diff
@@ -21,3 +14,25 @@ cprover_default_properties(
     java-unit
     miniz
 )
+
+# java models library
+find_program(MAVEN_PROGRAM mvn REQUIRED)
+
+file(GLOB_RECURSE java_models_sources "lib/java-models-library/src/*.java")
```

Comment:
FWIW if ergonomics is a concern then I wouldn't recommend using globs. The problem is that they're only re-evaluated on a reconfigure, so if you add another file it won't be picked up, and removing a file is an error. This is particularly annoying when switching branches, or when doing a git-bisect.

See also [this SO Q&A](https://stackoverflow.com/questions/1027247/is-it-better-to-specify-source-files-with-glob-or-each-file-individually-in-cmak).

Now I realise I'm kind of late to this because we already use globbing elsewhere, and the models library doesn't change that much anyway, so not a blocker, I'll just mention this whenever I see this.

Reply:
Yep, I'm regularly running into this with `git-bisect`. But I think that ship has sailed, we're doing it all over the place, and indeed I'm least concerned with the slow-moving target that the java models library is. So if we, at some point, decide to expand all the globs into explicit list then of course we should do this here as well. But for now it seems we can happily use globbing without making things much worse...